frontend/packages/dev-console/features/pipeline-Add_pages_for_tekton_trigger

```diff
@@ -0,0 +1,89 @@
+Feature: Add page for Tekton Trigger
+	As a user, I want see a page for Tekton Triggers
+
+Background:
+    Given user logged into the openshift application
+    And Pipelines operator should be installed
+
+@regression, @smoke
+Scenario: Adding triggers from developer view
+   Given User is in pipeline section in developer view having a pipeline
+   When User click on kebab menu of the pipeline
+   And  User selects Add Trigger
+   Then Add Trigger dialog box opens
+   And  Add Git Provider Type as github-pullreq 
+   And  User provide git resource and image resources if required
+   And  User clcks on Add
+   And  User go to pipeline detail page  
+   Then  User can see Trigger template and event listner 
+
+
+@regression, @smoke
+Scenario: Remove triggers from developer view
+   Given User is in pipeline section in developer view having a pipeline with trigger associated
+   When User click on kebab menu of the pipeline
+   And  User selects Remove Trigger
+   Then Remove Trigger dialog box opens
+   And  User selects Trigger to be removed in the dropdown
+   And  User clicks Remove
+   And  User go to pipeline detail page  
+   And  User can not see Trigger template and event listner
+
+
+@regression
+Scenario: View Pipeline resources throughout the related types
+   Given User is in Admin view
+   When User clicks pipeline under Pipelines tab
+   And User navigate to Pipelines via the tabs on the top of the page
+   Then The Pipelines page contains the Pipelines Name, Last Run, Last Run Status, and Last Started columns
+   And  User navigate to Pipeline Runs via the tabs on the top of the page
+   And The Pipelines Runs page contains the Name, Status, Started, and Duration columns.
+   And User navigate to Pipeline Resources via the tabs on the top of the page
+   And The Pipelines Resources page contains the Name, Created, and Type columns.
+   And User navigate to Conditions via the tabs on the top of the page
+   And The Conditions page contains the Name and Created columns.
+
+@regression
+Scenario: Create option in pipeline page
+   Given User is in Admin view
+   When User clicks pipelines under Pipelines tab
+   And The user click on the Create button from top right corner.
+   Then The main action the user can take on the Pipelines page are create Pipelines, Pipeline Resources and Conditions
```

Comment:
Split it in multiple Then statements like
Then user can see create pipelines action on the pipelines page
And user can see pipeline resources action on the pipelines page
And user can see conditions action on the pipelines page. 

It will be easy ti write step definitions of these statements.